packages/core-commands/src/index.js

```diff
@@ -1 +1,35 @@
-export { privateApis } from './private-apis';
+/**
+ * Internal dependencies
+ */
+import { unlock } from './lock-unlock';
+import { privateApis } from './private-apis';
+export { privateApis };
+
+/**
+ * WordPress dependencies
+ */
+import { createRoot } from '@wordpress/element';
+import { CommandMenu } from '@wordpress/commands';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+const { useCommands } = unlock( privateApis );
+const { RouterProvider } = unlock( routerPrivateApis );
+
+const mountPoint = document.createElement( 'div' );
+mountPoint.id = 'wp-commands';
+
+document.body.appendChild( mountPoint );
+
+const root = createRoot( mountPoint );
+
+function CommandMenuWrapper() {
+	useCommands();
+
+	return (
+		<RouterProvider>
+			<CommandMenu />
+		</RouterProvider>
+	);
+}
+
+root.render( <CommandMenuWrapper /> );
```

Comment:
While I like this way of rendering the command menu everywhere, I'm not sure that the "core-commands" package whose sole purpose is to have a list of commands that works cross pages. Would a small inline script work instead of a package for this?

Reply:
I'm having second thoughts here, so I'd love more opinions here @WordPress/gutenberg-core

Reply:
I agree it doesn't seem the right approach but the was a circular dependency if I implemented this inside of the command component directly. So perhaps it needs to be it's own component loader completely.

Reply:
I'll try the approach of the inline script. Good idea :)

Reply:
Inline script done in [2873f18](https://github.com/WordPress/gutenberg/pull/54515/commits/2873f18de2b099f047274eca2d754fa7144c89a0)